sherlock/sites.py

```diff
@@ -171,6 +175,18 @@ def __init__(self, data_file_path=None):
 
         return
 
+    def remove_nsfw_sites(self):
+        """
+        Remove NSFW sites from the sites, if isNSFW flag is true for site
+
+        Keyword Arguments:
+        self                   -- This object.
+
+        Return Value:
+        None
+        """
+        self.sites = {site:self.sites[site] for site in self.sites if not self.sites[site].is_nsfw }
```

Comment:
List comprehension is cool, but may make it difficult for newcomers to understand the code. They may perceive it as very complicated.

So please decompress this and make this more legible